oblib/taxonomy_semantic.py

```diff
@@ -182,37 +182,39 @@ def _load_concepts_file(self, pathname):
     def _load_concepts(self):
         """Return a dict of available concepts."""
         concepts = {}
-        for dirname in os.listdir(os.path.join(constants.SOLAR_TAXONOMY_DIR,
-                                               "data")):
-            for filename in os.listdir(
-                    os.path.join(constants.SOLAR_TAXONOMY_DIR, "data",
-                                 dirname)):
-                # if 'def.' in filename:
-                if 'pre.' in filename:
-                    concepts[dirname] = self._load_concepts_file(
-                        os.path.join(constants.SOLAR_TAXONOMY_DIR,
-                                     "data", dirname, filename))
-        for dirname in os.listdir(os.path.join(constants.SOLAR_TAXONOMY_DIR,
-                                               "documents")):
-            for filename in os.listdir(
-                    os.path.join(constants.SOLAR_TAXONOMY_DIR, "documents",
-                                 dirname)):
-                # if 'def.' in filename:
-                if 'pre.' in filename:
-                    concepts[dirname] = self._load_concepts_file(
-                        os.path.join(constants.SOLAR_TAXONOMY_DIR,
-                                     "documents", dirname, filename))
-
-        for dirname in os.listdir(os.path.join(constants.SOLAR_TAXONOMY_DIR,
-                                               "process")):
-            for filename in os.listdir(
-                    os.path.join(constants.SOLAR_TAXONOMY_DIR, "process",
-                                 dirname)):
-                # if 'def.' in filename:
-                if 'pre.' in filename:
-                    concepts[dirname] = self._load_concepts_file(
-                        os.path.join(constants.SOLAR_TAXONOMY_DIR,
-                                     "process", dirname, filename))
+        # for dirname in os.listdir(os.path.join(constants.SOLAR_TAXONOMY_DIR,
+        #                                        "data")):
+        for filename in os.listdir(
+                os.path.join(constants.SOLAR_TAXONOMY_DIR, "data")):
+            # if 'def.' in filename:
+            if 'pre.' in filename:
+                concept_name = filename[filename.find("solar-")+6:filename.find("_2019")]
+                if concept_name == "cutsheet":   # Note: CutSheet is not named using camel case like other files
```

Comment:
suggest `concept_name.lower() == "cutsheet"` so we don't have to change this if the taxonomy changes to camel case name

Reply:
The Taxonomy is CamelCase but there is a bug in cutsheet that the file name is lower case.  Jessie and I agreed to the work around (there is other issues with renaming the file right now) and continuing the discussion at the next work group meeting.